package.json

```diff
@@ -4,12 +4,14 @@
     "description": "Easily create CSV data from json collection",
     "repository": {
         "type": "git",
-        "url": "https://github.com/alexcaza/export-to-csv.git"
+        "url": "git+https://github.com/commoditrader/export-to-csv.git#fd88cd5"
     },
     "scripts": {
         "dev": "tsc --watch",
         "test": "tsc && karma start",
-        "prepublishOnly": "tsc"
+        "prepublishOnly": "tsc",
+        "build": "tsc",
+        "postinstall": "npm run build"
```

Comment:
Allow to run that command on post-install phase, npm can run directly  `npm run tsc` but can do that using the other scripts for some reason. There is a topic about that on the WEB